homeassistant/components/config/customize.py

```diff
@@ -0,0 +1,45 @@
+"""Provide configuration end points for Customize."""
+import asyncio
+
+from homeassistant.components.config import EditKeyBasedConfigView
+from homeassistant.components import SERVICE_RELOAD_CORE_CONFIG
+from homeassistant.config import DATA_CUSTOMIZE
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.core import DOMAIN as CORE_DOMAIN
+
+CONFIG_PATH = 'customize.yaml'
+
+
+@asyncio.coroutine
+def async_setup(hass):
+    """Set up the Customize config API."""
+    @asyncio.coroutine
+    def hook(hass):
```

Comment:
Can you add the async version and import it from `homeassistant.components` (it already has the sync version of this method)

Reply:
Done